### PR TITLE
Frontend/feat: #2464 Added support for global config

### PR DESCRIPTION
**Pull request by Vignesh-772**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

Added support for Global Config

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-common/src/Engineering/Helpers/FileProvider/Utils.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+export const loadCommonConfig = function () {
+  return JSON.stringify(defaultConfig);
+}
```
Comment (ashishsingh101) [inline Frontend/ui-common/src/Engineering/Helpers/FileProvider/Utils.js:3]: are we using this function ?

Comment (ashishsingh101) [reply Frontend/ui-common/src/Engineering/Helpers/FileProvider/Utils.js:3]: and from where the `defaultConfig ` is coming

File: Frontend/ui-common/src/Engineering/Strings/Strings.purs [hunk 2]
```diff
@@ -12,6 +12,19 @@
...
+configuration_file = "v1-configuration"
+
+dotJSA :: String
+dotJSA = ".jsa"
+
+dotJS :: String
+dotJS = ".js"
+
+dotJSON :: String
+dotJSON = ".json"
```
Comment (ashishsingh101) [inline Frontend/ui-common/src/Engineering/Strings/Strings.purs:30]: can we replace Strings with Constant

**Review by ashishsingh101 — CHANGES_REQUESTED** [review]

File: Frontend/ui-common/src/Engineering/Helpers/FileProvider/Utils.js (hunk 1 above)
Comment (Vignesh-772) [reply Frontend/ui-common/src/Engineering/Helpers/FileProvider/Utils.js:3]: It is not used. Removing

File: Frontend/ui-common/src/Engineering/Helpers/FileProvider/Utils.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        }
+      });
+    }
+  }
+  return finalObject;
+}
+
+export const loadInWindow = function (key, value) {
+  try { // Checking whether it is a Stringified JSON
+    window[key] = JSON.parse(value);
```
[the remark this answers is not in the document]
Comment (Vignesh-772) [reply Frontend/ui-common/src/Engineering/Helpers/FileProvider/Utils.js:53]: it is not checking whether string or not. it is converting it into JSON if it is stringified. typeof will not be helpful here.

**Review by ashishsingh101 — APPROVED** [review]

**Pull request merged**